app/Services/UserService.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Http\Request;
+
+class UserService
+{
+    public function getUser(Request $request): User
+    {
+        return \Auth::user();
```

Comment:
Fair nok, men er det ikke lidt overkill, hva? 😁

Reply:
Det er for at samle kaldet i et sted, for ellers vil vi skulle ændre 100 steder, hvis vi kommer til at ville justere noget.